charts/nexus3/templates/job-config.yaml

```
@@ -25,16 +25,10 @@ spec:
         - name: config
           image: {{ include "nexus3.image" .Values.config.job.image }}
           imagePullPolicy: {{ .Values.config.job.image.pullPolicy }}
+          {{- with .Values.nonRootSecurityContext }}
```

```suggestion
          {{- with .Values.config.job.securityContext }}
```

I think this should be job specific. Could you also add `config.job.podSecurityContext` and implement that here using the same defaults as the stateful set?